springmvc-raml-parser/src/test/java/com/phoenixnap/oss/ramlapisync/generation/rules/Issue211RulesTest.java

```diff
@@ -0,0 +1,35 @@
+
+package com.phoenixnap.oss.ramlapisync.generation.rules;
+
+import com.phoenixnap.oss.ramlapisync.data.ApiResourceMetadata;
+import com.phoenixnap.oss.ramlapisync.raml.InvalidRamlResourceException;
+import com.sun.codemodel.JCodeModel;
+import com.sun.codemodel.JDefinedClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.validation.Valid;
+
+public class Issue211RulesTest extends AbstractRuleTestBase {
+   JCodeModel cm = new JCodeModel();
+   private Rule<JCodeModel, JDefinedClass, ApiResourceMetadata> rule;
+   private ApiResourceMetadata controllerMetadata;
+
+   @BeforeClass
+   public static void initRaml() throws InvalidRamlResourceException {
+//      AbstractRuleTestBase.RAML = RamlLoader.loadRamlFromFile("ISSUE-211-Create-Update-naming-resolution.raml");
+//      AbstractRuleTestBase.RAML = RamlLoader.loadRamlFromFile("raml08issue.raml");
+      AbstractRuleTestBase.RAML = RamlLoader.loadRamlFromFile("raml10NoIssue.raml");
+   }
+
+   @Test
+   public void applySpring4ControllerInterfaceRule_shouldCreate_validCode() throws Exception {
+       jCodeModel = new JCodeModel();
+       cm = new JCodeModel();
+      rule = new Spring4ControllerInterfaceRule();

```

Comment:
At this point jCodeModel contain 3 packages and 17 refClasses, while cm only contains 0 and 9 respectively.

Reply:
![image](https://user-images.githubusercontent.com/11991425/33771031-412de058-dc38-11e7-95f6-49a21e348c5e.png)

Reply:
I also get "Skipped breakpoint at ..JCodeModel because it happens inside debugger evaluation" from time to time, as well in some other places, so I was thinking there might be something wrong with Intellij Idea. It also gives me false positives from time to time in my tests, and I need to restart it.
![image](https://user-images.githubusercontent.com/11991425/33771332-5763c152-dc39-11e7-8371-1d3664750968.png)